src/System.Runtime.Serialization.Formatters/tests/SerializationTypes.cs

```diff
@@ -78,23 +81,40 @@ public override bool Equals(object obj)
     }
 
     [Serializable]
-    public class Point
+    public class Point : IComparable<Point>, IEquatable<Point>
     {
         public int X;
         public int Y;
 
+        public Point()
```

Comment:
Question: why adding this public default ctor? This may have changed the test scenario. Before the change, the test was for type without public default ctor; after the fix, the test is for type having public constructor. DCS deal with these two cases differently. I'm not sure how BinaryFormatter would deal with this.